tests/test_organization.py

```diff
@@ -192,34 +192,37 @@ def test_organization_edit_buttons(setUp,
                                    full_organizations,
                                    membership_person,
                                    membership_organization):
-     org = full_organizations[0]
-     composition = Composition.objects.first()
-     person = org.personnel[0]
-     association = org.associations[0]
-     emplacement = org.emplacements[0]
-
-     assert is_tab_active(setUp.get(reverse_lazy('edit-organization', args=[org.uuid])),
-                          'Basics') == True
-
-     assert is_tab_active(setUp.get(reverse_lazy('create-organization-composition', args=[org.uuid])),
-                          'Relationships') == True
-     assert is_tab_active(setUp.get(reverse_lazy('edit-organization-composition', args=[org.uuid, composition.pk])),
-                          'Relationships') == True
-
-     assert is_tab_active(setUp.get(reverse_lazy('create-organization-personnel', args=[org.uuid])),
-                          'Personnel') == True
-     assert is_tab_active(setUp.get(reverse_lazy('edit-organization-personnel', args=[org.uuid, person.pk])),
-                          'Personnel') == True
-
-     assert is_tab_active(setUp.get(reverse_lazy('create-organization-emplacement', args=[org.uuid])),
-                          'Locations') == True
-     assert is_tab_active(setUp.get(reverse_lazy('edit-organization-emplacement', args=[org.uuid, emplacement.pk])),
-                          'Locations') == True
-
-     assert is_tab_active(setUp.get(reverse_lazy('create-organization-association', args=[org.uuid])),
-                          'Locations') == True
-     assert is_tab_active(setUp.get(reverse_lazy('edit-organization-association', args=[org.uuid, association.pk])),
-                          'Locations') == True
-
-     assert is_tab_active(setUp.get(reverse_lazy('create-organization-membership', args=[org.uuid])),
-                          'Relationships') == True
+    org = full_organizations[0]
+    composition = Composition.objects.first()
+    person = org.personnel[0]
+    association = org.associations[0]
+    emplacement = org.emplacements[0]
+    membership_org = membership_organization.organization.get_value()
```

Comment:
`membership_organization.organization.get_value().pk` and `membership_organization.pk` will actually return the same value, so we don't really need this variable. This is because `MembershipOrganization.organization.get_value()` returns a `MembershipOrganizationMember` object, which is a complex field relating two organizations:

https://github.com/security-force-monitor/sfm-cms/blob/c5a163aee02ab1db810d3760c2b9c7c431bcad35/membershiporganization/models.py#L55-L63

This complex field will have the same `pk` attribute as its parent `MembershipOrganization`.

Adjusting the call here is optional, I just wanted to take the opportunity to highlight a particular feature of complex fields!

Reply:
for consistency, i will change it to `membership_organization.pk`.